crates/router/src/connector/globalpay/transformers.rs

```diff
@@ -149,6 +149,28 @@ impl From<GlobalpayPaymentStatus> for enums::RefundStatus {
     }
 }
 
+fn get_payment_response(
+    status: enums::AttemptStatus,
+    response: GlobalpayPaymentsResponse,
+) -> Result<types::PaymentsResponseData, ErrorResponse> {
+    match status {
+        enums::AttemptStatus::Failure => Err(ErrorResponse {
+            message: response
+                .payment_method
+                .and_then(|pm| pm.message)
+                .map_or("".to_string(), |f| f),
```

Comment:
There are constant for no message present, avoid using empty strings. Instead map_or, or will work. So that we don't want to use closure just to return the same input.